README.md

```diff
@@ -36,7 +42,16 @@ signalfx-java will support this.
 </dependency>
 ```
 
-### From source (Not recommended)
+To install this library in Scala, add the following to your project's
+`build.sbt` file.
```

Comment:
SBT is not the only build tool for Scala. Maven can be used as well, as described above. I'm sure Gradle has again a different syntax. My point is this section should be structured by build tool, not by language.